AIDevGallery.SourceGenerator/Models/HardwareAccelerator.cs

```diff
@@ -10,5 +10,6 @@ internal enum HardwareAccelerator
 {
     CPU,
     DML,
-    QNN
+    QNN,
+    WCRAPI
```

Comment:
Is this logic specific for WCRAPIs or for all future APIs (Azure, OAI, etc)?

Reply:
just for wcrapis